src/main/assets/js/index.ts

```diff
@@ -1,7 +1,13 @@
+import { initAll } from 'govuk-frontend';
+
 import '../scss/main.scss';
 import './go-back';
 import './session-timeout';
 import './enter-address';
-import { initAll } from 'govuk-frontend';
+import { UPLOAD_YOUR_DOCUMENTS } from '../../steps/urls';
+
+if (document.location.pathname === UPLOAD_YOUR_DOCUMENTS) {
```

Comment:
isn't this bundled at build time?

Reply:
This get converted in a special inlined  `__webpack_require__`, but I've moved this check into the file itself to be consistent with how the other import work